test/UserOp.ts

```diff
@@ -83,7 +77,10 @@ export const DefaultsForUserOp: UserOperation = {
   preVerificationGas: 21000, // should also cover calldata cost.
   maxFeePerGas: 0,
   maxPriorityFeePerGas: 1e9,
-  paymasterAndData: '0x',
+  paymaster: '0x',
```

Comment:
AddressZero (or left unk, if logic of "fill" handles it)

Reply:
Does it matter?

Reply:
you assume the encoder accepts "0x" as a valid address... we try to reduce implicit assumptions and stick to the actual type.
(it passes compliation only because typescript currently doesn't differentiate "address" type from "bytes" type)